### PR TITLE
Files transfer 

### PR DESCRIPTION
**Pull request by Bounce23**

### CODE REVIEW

File: sage_codes/Safe Curve code [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+  safecurve &= requirement('verify-safecomplete',safecomplete)
+  safecurve &= requirement('verify-safeind',safeind)
+  requirement('verify-safecurve',safecurve)
+
+originaldir = os.open('.',os.O_RDONLY)
+for i in range(1,len(sys.argv)):
+os.fchdir(originaldir)
+os.chdir(sys.argv[i])
+verify()
+
```
Comment (CPerezz) [inline sage_codes/Safe Curve code]: Perhaps add two or three \n to differentiate better both sections.

File: src/backend/mod.rs [hunk 2]
```diff
@@ -1,17 +1,18 @@
-//! Contains the different backend implementations: `u64` and further comming ones.
+
```
Comment (CPerezz) [inline src/backend/mod.rs]: Remove that blank line. Doc comments need to go on the first line.

File: src/backend/u64/field.rs [hunk 3]
```diff
@@ -278,6 +277,7 @@ impl Mul<FieldElement> for FieldElement {
     }
 }
 
+
```
Comment (CPerezz) [inline src/backend/u64/field.rs]: Only one blank line between implementations.

File: src/backend/u64/field.rs [hunk 4]
```diff
@@ -305,6 +305,7 @@ impl Div<FieldElement> for FieldElement {
...
```
Comment (CPerezz) [inline src/backend/u64/field.rs]: Only one blank line between implementations.

File: src/backend/u64/field.rs [hunk 5]
```diff
@@ -524,6 +524,7 @@ impl FieldElement {
     /// Load a `FieldElement` from the low 253b bits of a 256-bit
     /// input. So Little Endian representation in bytes of a FieldElement.
     // @TODO: Macro for Inline load8 function as it has variadic arguments.
+    #[warn(dead_code)]
```
Comment (CPerezz) [inline src/backend/u64/field.rs]: Why is this added? from_bytes() is a function called in the lib already, so it shouldn't need to use the warning disablement.

Comment (Bounce23) [reply src/backend/u64/field.rs]: As the ultimate goal is for complete Ristretto injection, I figured showing how the curve code can work using this could be of importance. To look at factors such as the prime field as a whole and give a justification of why they are set (in our case, ristretto), then insight into the functions can be useful.

**Comment by CPerezz** [comment]
Why was this merged without fixing the requested changes?
